src/helpers/path.js

```diff
@@ -1,16 +1,19 @@
-const join = require('join-path')
-const fs = require('fs')
 const _ = require('lodash/fp')
+const fs = require('fs')
+const join = require('join-path')
 const assert = require('assert')
 const normalize = _.replace(/^\/+|$/g, '')
 const parts = _.pipe(normalize, _.split('/'))
+
+
```

Comment:
did the linter require 2 blank lines here?

Reply:
mayvbe this was me :) fixed!